packages/kcms/src/match/model/main.ts

```diff
@@ -13,10 +14,21 @@ export interface CreateMainMatchArgs {
   teamId2?: TeamID;
   winnerId?: TeamID;
   runResults: RunResult[];
+  parentMatchID: MainMatchID | undefined;
+  childMatches: ChildrenMatches | undefined;
+}
+
+/**
+ * 自分より前に行われた試合
+ */
```

Comment:
もうちょっとマシなコメントをください(前に行われた試合全部を指すように読めます)
```suggestion
/**
 * トーナメントで自分の1つ前の2試合
 */
```
とか？